Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM rust:1.51.0 AS build
+
+COPY . /usr/src/gleam-src
+
+RUN set -xe \
+        && cd /usr/src/gleam-src \
+        && make install \
+        && rm -rf /usr/src/gleam-src
+
+FROM node:15.14.0
+
+COPY --from=build /usr/local/cargo/bin/gleam /bin
+RUN gleam --version
+
+WORKDIR /opt/app
+CMD ["bash"]
```

Comment:
This is just a Dockerfile that lives in my repo while I'm developing it. I'm sharing it here in case it's helpful for anyone debugging the errors I get.

Currently I'm running
```
gleam compile-package --src . --out gen --name tessa --target javascript
// then
sirv . --host 0.0.0.0 --dev
```

@lpil is there a watch option for `gleam compile-package` (or quick hack) you could think of so I can just change my gleam code and refresh the page and get the updated JS?

Reply:
I use `watchexec` so it's `watchexec -- gleam compile-package`, or whatever command you want to run

https://github.com/watchexec/watchexec